designs/dragonphy_top/constraints/gen_constraints.py

```diff
@@ -233,6 +231,66 @@
 }}]
 set_dont_touch_network $tdbg_clk_pins
 
+# TODO: do we need to set dont_touch through the hierarchy?
```

Comment:
This is where I set ``dont_touch`` on nets and cells in the transmitter.  I experimented with applying ``dont_touch`` at various levels, to nets, pins, and cells.  It may be that there is a simpler way to apply the constraint; I haven't tried to trim down the code yet.